3.6/conf.d/apache.conf

```diff
@@ -1,13 +1,13 @@
 Listen 8080
 <VirtualHost *:8080>
-        DocumentRoot /app/src/build/website
-        CustomLog /var/log/apache2/access.log combined
-        ErrorLog /var/log/apache2/error.log
-        LogLevel debug
-        <Directory /app/src/build/website>
-                Options FollowSymLinks MultiViews
-                DirectoryIndex search.php
-                Require all granted
-        </Directory>
-        AddType text/html .php
+  DocumentRoot /nominatim/website
```

Comment:
This doesn't look right. Are you sure that this is the correct path?

Looks like you copied the 3.7 version to 3.6 and 3.5

Reply:
🤦🏻‍♂️


thanks.